packages/shopify-codemod/transforms/iife-to-ternary-expression.js

```diff
@@ -0,0 +1,48 @@
+export default function iifeToTernaryExpression({source}, {jscodeshift: j}, {printOptions = {}}) {
+  function isImmediatelyInvokedFunctionExpression(expressionPath) {
+    let expression = expressionPath.node.expression
+    return expression.type === 'CallExpression' && expression.callee.type === 'ArrowFunctionExpression'
+  }
+
+  function containsOnlyIfStatement(functionNode) {
+    return nodeIsBlockStatementContainingOnly(functionNode.body, 'IfStatement');
+  }
+
+  function onlyReturnsOneOfTwoValues(ifStatementNode) {
+    return nodeIsBlockStatementContainingOnly(ifStatementNode.consequent, 'ReturnStatement') &&
+           nodeIsBlockStatementContainingOnly(ifStatementNode.alternate, 'ReturnStatement');
+  }
+
+  function nodeIsBlockStatementContainingOnly(node, childType) {
+    return node.type === 'BlockStatement' &&
+           node.body.length === 1 &&
+         node.body[0].type === childType;
+  }
+
+  function isConvertibleExpression(path) {
+    return isImmediatelyInvokedFunctionExpression(path) &&
+           containsOnlyIfStatement(path.node.expression.callee) &&
+           onlyReturnsOneOfTwoValues(path.node.expression.callee.body.body[0]);
+  }
+
+  const sourceAST = j(source);
+
+  sourceAST
+    .find(j.ExpressionStatement)
```

Comment:
I highly recommending running `esify` at least once on `shopify/shopify`.  It'll give you a feel for the impact you're making on the code (and if you're not, you can say valuable things like, "this transform isn't worth the effort").

Here's the gonzo script that I use to update `esify` to contain my work-in-progress transforms:
```sh
node_version=v5.7.0
global_esify_root="/Users/Gord/.nvm/versions/node/$node_version/lib/node_modules/esify"

cp -R ~/shopify/javascript/packages/shopify-codemod/transforms/** $global_esify_root/node_modules/shopify-codemod/transforms
cp ~/shopify/javascript/packages/esify/bin/esify "$global_esify_root/bin/esify"
cp -R ~/shopify/javascript/packages/esify/lib "$global_esify_root"

rm -fr "$global_esify_root/node_modules/shopify-decaf"
cp -R ~/shopify/decaf/ "$global_esify_root/node_modules/shopify-decaf"
```